File: nga/app/src/main/java/org/gosky/nga/data/entity/HttpBaseResult.java
package org.gosky.nga.data.entity;

/**
 * Created by hunji on 2016/9/1.
 * desc:
 */
public class HttpBaseResult<T> {

    /**
     * server_time : 1477284985
     * success : true
     * message :
     * data :
     */

    private int server_time;
    private boolean success;
    private String message;
    private T data;

    public int getServer_time() {
        return server_time;
    }

    public void setServer_time(int server_time) {
        this.server_time = server_time;
    }

    public boolean isSuccess() {
        return success;
    }

    public void setSuccess(boolean success) {
        this.success = success;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }

    public T getData() {
        return data;
    }

    public void setData(T data) {
        this.data = data;
    }
}

File: nga/app/src/main/java/org/gosky/nga/data/entity/HttpBaseResult.kt
package org.gosky.nga.data.entity

/**
 * Created by hunji on 2016/9/1.
 * desc:
 */
class HttpBaseResult<T> {

    /**
     * server_time : 1477284985
     * success : true
     * message :
     * data :
     */

    var server_time: Int = 0
    var isSuccess: Boolean = false
    var message: String? = null
    var data: T? = null
}
